src/coreclr/inc/corinfo.h

```diff
@@ -189,8 +189,8 @@ TODO: Talk about initializing strutures before use
 #if !defined(__deref_inout_ecount)
 // Minimum set of SAL annotations so that non Windows builds work
 #define __deref_inout_ecount(size)
```

Comment:
Missed find&replace?

Reply:
Thanks.
I see that I didn't update __deref** annotations.